web/client/plugins/tocitemssettings/defaultSettingsTabs.js

```diff
@@ -150,8 +151,11 @@ const getConfiguredPlugin = (plugin, loaded, loadingComp) => {
 export const getStyleTabPlugin = ({ settings, items = [], loadedPlugins, onToggleStyleEditor = () => { }, onUpdateParams = () => { }, element, ...props }) => {
 
     if (isVectorStylableLayer({element})) {
+        const ConnectedVectorStyleEditor = connect(
+            createSelector([isCesium], ({ enable3dStyleOptions }) => ({ enable3dStyleOptions }))
+        )(VectorStyleEditor);
```

Comment:
Please move the connected component outside the function, I think this getStyleTabPlugin is triggered multiple time during the render and the connect can be applied only once at the initialization